sdk/README.md

```diff
@@ -33,13 +33,13 @@ a basic sequential pipeline.
 
 4. Compile the sample pipeline:
 
-    - `cd sdk/samples/sequential`  
-    - `dsl-compile-tekton --py ./sequential.py --output pipeline.yaml`
+    - `cd sdk/samples`  
+    - `dsl-compile-tekton --py ./parallel_and_sequential.py --output pipeline.yaml`
```

Comment:
lets just call it `parallel_join.py` even though it does imply the join step following both parallel steps. Once we add the parameter passing from #19 then the order of tasks will be implied and enforced by the Tekton runtime.

Reply:
thanks, done